src/introspection.rs

```diff
@@ -92,10 +85,13 @@ pub trait DynamicIntrospection {
     where
         I: IntoIterator<Item = Interface> + Send;
 
+    /// Add one or more [`Interface`] to the device introspection, specialized for a [`Vec`].
```

Comment:
I suggest you to change the doc comment at line 80 and make it similar to the ones at line 83 and 89. It is not an error, however it may be misleading (as if the 2 methods are doing different things whereas they are both adding 1 or more `Interface`s)